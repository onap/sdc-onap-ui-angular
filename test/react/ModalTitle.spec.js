import React from 'react';
import ModalTitle from '../../src/react/ModalTitle.js';

import renderer from 'react-test-renderer';

describe('ModalTitle', () => {
	test('basic test', () => {
	    const header = renderer.create(<ModalTitle>Title</ModalTitle>).toJSON();        
	    expect(header).toMatchSnapshot();
	});
});