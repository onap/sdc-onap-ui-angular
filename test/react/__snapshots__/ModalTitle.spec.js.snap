// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalTitle basic test 1`] = `
<div
  className="title "
>
  Title
</div>
`;
